### DISCUSSION TITLE
[Articles] How to build better Machine Learning Models

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [EddieHubCommunity/support#1411] [Show and tell]
My new blog is out! If you are interested or learning about Machine Learning you might find this one interesting. This article talks about a couple of precise and practical tips and guidelines to help you build better Machine Learning Models. Consider giving it a clap :clap: if you like it:

https://medium.com/@rishit.dagli/how-to-build-better-machine-learning-models-537a4486d056

**Comment by Prathamesh-Shanbhag** [comment]
Hey, Rishit from what I read you definitely, know your stuff about Machine Learning! Although I'm not really familiar with Machine Learning Models too well, if there's a link to a blog for ML for beginners that you've written and would like to share, I would like to give it a read as well!

**Reply by Rishit-dagli** [reply]
Hi @Prathamesh-Shanbhag ,
Thank You.
Sure I had started this short blog series for folks starting with practical ML: https://medium.com/@rishit.dagli/get-started-with-tensorflow-and-deep-learning-part-1-72c7d67f99fc
